spec.emu

```diff
@@ -16,17 +16,13 @@ contributors: Richard Gibson
   <p>The abstract operation QuoteJSONString with argument _value_ wraps a String value in QUOTATION MARK code units and escapes certain other code units within it.</p>
   <emu-alg>
     1. Let _product_ be the String value consisting solely of the code unit 0x0022 (QUOTATION MARK).
-    1. For each code unit _C_ in _value_, do
-      1. If the numeric value of _C_ is listed in the Code Unit Value column of <emu-xref href="#table-json-single-character-escapes"></emu-xref>, then
+    1. For each code <del>unit</del><ins>point</ins> _C_ in _value_ <ins>when interpreted as UTF-16 encoded Unicode text as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, do
+      1. If <del>the numeric value of</del> _C_ is listed in the Code <del>Unit Value</del><ins>Point</ins> column of <emu-xref href="#table-json-single-character-escapes"></emu-xref>, then
         1. Set _product_ to the string-concatenation of _product_ and the Escape Sequence for _C_ as specified in <emu-xref href="#table-json-single-character-escapes"></emu-xref>.
-      1. Else if _C_ has a numeric value less than 0x0020 (SPACE), then
+      1. Else if _C_ has a numeric value less than 0x0020 (SPACE), <ins>or _C_ has the same numeric value as a <emu-xref href="#leading-surrogate-code-unit"></emu-xref> or <emu-xref href="#trailing-surrogate-code-unit"></emu-xref>,</ins> then
         1. Set _product_ to the string-concatenation of _product_ and UnicodeEscape(_C_).
```

Comment:
UnicodeEscape is documented to accept a code unit, not a code point, so either that operation or invocations thereof need to updated as well.